x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/ConnectorIndexService.java

```diff
@@ -166,6 +169,36 @@ public void onFailure(Exception e) {
         }
     }
 
+    /**
+     * Updates the {@link ConnectorScheduling} property of a {@link Connector}.
+     *
+     * @param request  The request for updating the connector's scheduling.
+     * @param listener The listener for handling responses, including successful updates or errors.
+     */
+    public void updateConnectorScheduling(UpdateConnectorSchedulingAction.Request request, ActionListener<UpdateResponse> listener) {
+        try {
+            String connectorId = request.getConnectorId();
+            final UpdateRequest updateRequest = new UpdateRequest(CONNECTOR_INDEX_NAME, connectorId).doc(
+                new IndexRequest(CONNECTOR_INDEX_NAME).opType(DocWriteRequest.OpType.INDEX)
```

Comment:
I'm not sure if this applies as it's part of the update request, but I think the proper operation should be update. 

```suggestion
                new IndexRequest(CONNECTOR_INDEX_NAME).opType(DocWriteRequest.OpType.UPDATE)
```

Let's try this with implementing the 404 test mentioned above and see if it works correctly 👍

Reply:
That's what I thought as well - but the IndexRequest doesn't allow to `OpType.UPDATE` ([IndexRequest source code](https://github.com/elastic/elasticsearch/blob/714aa680f11514ee981682785423bc4d3bb7d383/server/src/main/java/org/elasticsearch/action/index/IndexRequest.java#L216)) - My understanding is that we are creating a new scheduling object (overwriting a field in a parent connector doc)

Reply:
Thanks for confirming! 👍